streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -781,4 +775,86 @@ public void shouldNotAllowToAddGlobalStoreWithSourceNameEqualsProcessorName() {
             sameNameForSourceAndProcessor,
             new MockProcessorSupplier());
     }
+
+    @Test
+    public void shouldThrowIfNameIsNull() {
```

Comment:
Needed to rewrite those test, because we use JUnit 4.12 in `2.2` branch (instead of `4.13-beta-2` as in `trunk`) that does not support `assertThrows`.

Upgrading JUnit to `4.13-beta-2` broke other tests so it's easier to stick with 4.12.